api/project/v1/project.proto

```diff
@@ -22,6 +22,9 @@ service ProjectAPI {
 message GetProjectsRequest {
   repeated string projects = 1;
   repeated string users = 2;
+  // If this is set to true, we will compute all project dependencies for the requested projects.
```

Comment:
by compute do you mean return? i'm not clear what this means.

Reply:
Yeah i guess return is a bit more clear, let me change this.